packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/platform_interface/google_maps_flutter_platform.dart

```diff
@@ -356,4 +356,38 @@ abstract class GoogleMapsFlutterPlatform extends PlatformInterface {
   }) {
     throw UnimplementedError('buildView() has not been implemented.');
   }
+
+  /// Returns a widget displaying the map view.
+  ///
+  /// This method is similar to [buildView], but contains a parameter for
+  /// platforms that require a text direction.
+  ///
+  /// Default behavior passes all parameters except `textDirection` to
+  /// [buildView].
```

Comment:
How about adding "This is for backward compatibility with existing implementations. Platforms that use the text direction should override this as the primary implementation, and delegate to it from `buildView`." That makes it clear why the default is (un-intuitively) to delegate in the direction that discards information, rather than the more usual pattern of the one with fewer arguments calling the other way with a default value.